internal/gengapic/custom_operation.go

```diff
@@ -130,6 +130,28 @@ func (g *generator) customOperationType() error {
 	p("}")
 	p("")
 
+	// Wait
+	p("// Wait blocks until the operation is complete, polling regularly")
+	p("// after an intial period of backing off between attempts.")
+	p("func (o *%s) Wait(ctx context.Context, opts ...gax.CallOption) error {", opName)
+	p("	 bo := gax.Backoff{")
+	p("    Initial: 1 * time.Second,")
```

Comment:
Are these numbers the same as for gRPC? It would probably make sense to define them centrally as constants rather than hard-coding them as literals into the templates. (though I'm fine with the generated code having literals)

Reply:
Yes they are the same and as gRPC. I'll put them in constants for the generator.

Reply:
I switched them both to drop the `1 *`, it is extraneous.